organization/_posts/2021/2021-03-18-coordination.md

```diff
@@ -0,0 +1,153 @@
+
+Apologies: Krzysztof Genser
+  
+## News, general matters, announcements
+
+### LHCC
+
+The software WLCG report for 2021H1 was prepared - see Indico agenda.
+
+Thanks to all who made comments and suggestions.
+
+### HL-LHC Review Phase 2
+
+We have had some meetings with key people for each section of the document. Now making progress in contacting other projects and arranging mini-workshops, as needed.
+
+### Google Summer of Code, Season of Docs
+
+The GSoC mentors are quite busy at the moment with candidate evaluation and there are few exchanges on the mentors list.
+
+For GSoD so far we have **no applications**. *Do no projects need documentation help?* If no proposals we won't apply. Deadline is 25 March.
+
+- Will send another reminder
+- Can we consider also facility areas that need documentation?
+    - Yes
+
+### November Workshop Survey and Outcomes
+
+See slides attached to the agenda, which give highlights of the survey from the workshop.
+
+- Feedback: no need for a single virtual workshop to be packed with disparate topics, so indeed perhaps better to split into smaller events.
+
+### Proposal for DUNE Framework Requirements Review
+
+Paul: DUNE is trying to workout how to deal with its data, including massive "events" from supernovae (100TB). DUNE task force has defined the requirements (43 of them) in a document, which is aimed at DUNE people. If the HSF is giving feedback on this, how can it be expanded/annotated to non-DUNE people? Once that's done can we get together a group of experts who could review and bring input from the expertise of the wider community?
+
+- Makes a lot of sense to have the HSF do this, crosscheck requirements and make sure that nothing is forgotten
+- Could we ask people from astro-particle to be involved as well, especially to look at the SN event issues
+
+Andrew: wanted to define a timescale on which feedback was given - early enough to be able to take requirements and statements back to FAs, viz. 4-6 weeks... (timescale is driven by US funding and a computing CDR that is due in June).
+
+- To crosscheck the requirements or to check how much existing software matches those (fixed) requirements? Would not be able to do both on a short timescale.
+
+Andrew: check on requirements should be a quick part; real question is how do those requirements fit into the HEP ecosystem?
+
+- We would try to leverage the broadest expertise possible, of people willing to help
+- Are the facility boundary conditions clear? GPUs or not? Dedicated or shared resources? Special facilities like analysis facilities?
+
+Andrew: there is a description of some of this and we do anticipate being quite free to decide how to use these resources; data sizes and form of a neutrino analysis are well known. Some feedback on this aspect would also be useful, when people come with different experience.
```

Comment:
```suggestion
Andrew: there is a description of some of this and we do anticipate being quite free to decide how to use these resources; data sizes and form of a neutrino analysis are well known. Some feedback on this aspect would also be useful, when people come with different experiences.
```

Reply:
Here I see "experience" as used in an abstract, uncountable, way - for me it's more natural like this.